blog/content/post/2019-04-12-bit-bang-theory-with-rte.md

```diff
@@ -0,0 +1,148 @@
+---
+title: The Bit Bang Theory with RTE
+cover: /covers/bit_bang_cover.jpg
+author: lukasz.wcislo
+layout: post
+published: true
+date: 2019-04-12
```

Comment:
`published: true` is okay, but the `date` should be set up for an exact scheduled date for publication